core/ibft_test.go

```diff
@@ -529,6 +539,18 @@ func TestRunNewRound_Proposer(t *testing.T) {
 							isValid,
 						)
 					},
+					getExtendedRCCFn: func(
+						height uint64,
+						isValidMessage func(message *proto.Message) bool,
+						isValidRCC func(round uint64, messages []*proto.Message) bool,
+					) []*proto.Message {
+						messages := filterMessages(
+							roundChangeMessages,
+							isValidMessage,
+						)
+
+						return messages
```

Comment:
Not a critical issue, but I believe it would be better if this had the same convention as https://github.com/0xPolygon/go-ibft/pull/52/files#diff-462fdf3f4e3d0ed0298c0ee42c7db641c69389c6ccedefb77428b1309da5880fR383

Reply:
Fixed 1df82823236c7a1b9b2706975258b9878ade77df